static/js/components/subject_page/disaster_event_block.tsx

```diff
@@ -296,6 +301,23 @@ function renderTiles(
           />
         );
       }
+      case "HISTOGRAM": {
+        return (
+          <HistogramTile
+            key={id}
+            id={id}
+            title={tile.title}
+            place={props.place}
+            selectedDate={getDate(props.id)}
+            disasterEventData={
+              disasterEventData[tile.histogramTileSpec.eventTypeKey] || {
+                eventPoints: [],
+                provenanceInfo: {},
```

Comment:
we should be handling these default values before this (i.e. disasterEventData should have reasonable defaults for all event types). or the component should handle the case where disasterEventData is null. of these, i might choose (2)

Reply:
Went with (2). Added handling so that histogram just doesn't render if disasterEventData is null instead of breaking the entire component.